schemas/preferences.sublime-settings.json

```
@@ -165,6 +165,11 @@
           "type": "string",
           "default": "./\\()\"'-:,.;<>~!@#$%^&*|+=[]{}`~?"
         },
+        "sub_word_separators": {
+          "markdownDescription": "Characters that are considered to separate subwords.",
```

```suggestion
          "markdownDescription": "Characters that are considered to separate sub-words.",
```

added a `-` just to match ST description for the setting

---

I just pushed a commit to address this but for some reason it never showed up here. Weird. Since it's a small change, I'll commit it here itself.

---

And for some reason, it now shows up. Firefox nightly is weirding out on me.